src/_pytest/compat.py

```diff
@@ -249,6 +249,21 @@ def get_real_func(obj):
     return obj
 
 
+def get_real_method(obj, holder):
```

Comment:
this needs to handle `__wrapped__` on python2

Reply:
Hmm `get_real_func` already handles `__wrapped__`. Can you clarify this a bit?

Reply:
@nicoddemus my understanding is that method descriptors are not exposing it readily, i will investigate in a bit